site/README.md

```diff
@@ -1,24 +1,26 @@
 # devswag.io
 
 ## About
-Source code for the https://devswag.io website.
+
+This folder contains the source code of the devSwag website (hosted at http://devswag.io).
```

Comment:
Use HTTPS

Reply:
How about renewing the certificate when it expires? :)

Reply:
I agree. 😂 
It's mostly a matter of more control (one-way door vs two-way doors). If you have strong reason against it, please share and we can discuss that. 💯

Reply:
Netlify is configured to autorenew the certificate, right ? If yes, I recommend we force https as the cert will always be valid.

Reply:
you might need acme.sh

Reply:
Please dont, netlify offers it integration: https://www.netlify.com/docs/ssl/#netlify-certificates